libs/chatbot/lib/components/ChatBot/helpers.ts

```
@@ -37,10 +40,65 @@ export const getUserQuestionForBotAnswer = (
+      } catch {
+        return undefined;
+      }
+
+      const url = res?.length ? res[0].url : undefined;
+      if (url) {
+        return {
+          title: 'Download ISO',
+          url,
+        };
+      }
+    }
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)